scripts/src/javatests/org/oppia/android/scripts/docs/KdocValidityCheckTest.kt

```diff
@@ -237,6 +237,37 @@ class KdocValidityCheckTest {
     assertThat(outContent.toString().trim()).isEqualTo(KDOC_CHECK_PASSED_OUTPUT_INDICATOR)
   }
 
+  @Test
+  fun testKdoc_functionWithBindsAnnotation_withoutKdoc_checkShouldPass() {
```

Comment:
Can you please add screenshots in the opening comment showing both of these tests failing without the fix in place? (This is generally good practice since it ensures the tests correctly catch the issue we want to prevent with the fix).

Reply:
Done.